apps/src/StudioApp.js

```diff
@@ -508,7 +508,7 @@ StudioApp.prototype.assetUrl_ = function (path) {
     throw new Error('StudioApp BASE_URL has not been set. ' +
       'Call configure() first');
   }
-  return this.BASE_URL + path;
+  return this.BASE_URL + (digestManifest[path] || path);
```

Comment:
`readonly.html.ejs` uses `assetUrl` to load Blockly in an iframe for feedback blocks.  Really the feedback blocks should be a new `BlockSpaceEditor` instance instead of a separate view, but I'm trying to keep the scope of this change small :smile:.